### PR TITLE
Adding Old Parquet Format Releases with Links

### PR DESCRIPTION
**Pull request by vinooganesh**
Updating to add more parquet format releases

@wgtmac

### CODE REVIEW

File: content/en/blog/parquet-format/2.10.0.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: "2.10.0"
+date: 2023-11-20
+description: >
+---
+
+[Github Release Link](https://github.com/apache/parquet-format/releases/tag/apache-parquet-format-2.10.0).
+
+The [latest version of parquet-format is 2.9.0](https://www.apache.org/dyn/closer.lua/parquet/apache-parquet-format-2.10.0/apache-parquet-format-2.10.0.tar.gz).
```
Comment (wgtmac) [inline content/en/blog/parquet-format/2.10.0.md]: This looks strange to me.

File: content/en/blog/parquet-format/2.7.0.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+title: "2.7.0"
+date: 2019-09-25
+description: >
+---
+
+[Github Release Link](https://github.com/apache/parquet-format/releases/tag/apache-parquet-format-2.7.0).
+
+The [latest version of parquet-format is 2.7.0](https://www.apache.org/dyn/closer.lua/parquet/apache-parquet-format-2.7.0/apache-parquet-format-2.7.0.tar.gz).
```
Comment (wgtmac) [inline content/en/blog/parquet-format/2.7.0.md]: ditto

**Comment by vinooganesh** [comment]
@wgtmac updated to fix typo and new release url

File: content/en/blog/parquet-format/2.8.0.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+title: "2.8.0"
+date: 2020-01-13
+description: >
+---
+
+[Github Release Link](https://github.com/apache/parquet-format/releases/tag/apache-parquet-format-2.7.0).
```
Comment (wgtmac) [inline content/en/blog/parquet-format/2.8.0.md]: This page is not right.

File: content/en/blog/parquet-format/2.9.0.md [hunk 4]
```diff
@@ -4,31 +4,16 @@ date: 2021-10-06
 description: >
 ---
 
-The [latest version of parquet-format is 2.9.0](https://www.apache.org/dyn/closer.lua/parquet/apache-parquet-format-2.9.0/apache-parquet-format-2.9.0.tar.gz).
+[Github Release Link](https://github.com/apache/parquet-format/releases/tag/apache-parquet-format-2.7.0).
```
Comment (wgtmac) [inline content/en/blog/parquet-format/2.9.0.md]: ditto

**Comment by vinooganesh** [comment]
@wgtmac - Ugh sorry, I'm moving to vscode and was too used to the automatic file saving before commit in my intellij. Actually fixed now.

**Pull request merged**